Misc/NEWS.d/next/Library/2018-07-02-05-59-11.bpo-34019.ZXJIife.rst

```diff
@@ -0,0 +1,2 @@
+webbrowser: Fix the module opens Opera browser with wrong arguments.
```

Comment:
I find this NEWS entry a bit confusing. Maybe we can improve this with something in the lines of:

`Correct the arguments passed to the Opera browser when opening a new URL using the ``webbrowser`` module`

Reply:
@pablogsal Your phrases sounds much batter. My bad writing :P. I updated it.